### PR TITLE
Update osu!supporter (images, amount of hearts in a badge)

### PR DESCRIPTION
**Pull request by geshii**
o/
I updated the supporter page and also changed the images since these were not up to date

Should I add images to show the different badges ?

### CODE REVIEW

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Comment by geshii** [comment]
I think we are ready to merge, if there's nothing else to change

File: wiki/osu!supporter/en.md [hunk 1]
```diff
@@ -1,6 +1,3 @@
----
-outdated: true
```
Comment (TicClick) [inline wiki/osu!supporter/en.md:2]: you can't remove the outdated marker if you don't address the issues mentioned in TODO, and I don't feel that what you've changed so far is enough for that

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/osu!supporter/en.md (hunk 1 above)
Comment (geshii) [reply wiki/osu!supporter/en.md:2]: should I add the marker back or else what should I add to the page?

Comment (TicClick) [reply wiki/osu!supporter/en.md:2]: I've just skimmed through the article again and didn't find many things to improve. I think that for now you can replace `outdated: true` with `needs_cleanup: true`. 

@cl8n blame says the TODO is yours — any details on cleanup ideas other than replacing the images?

Comment (geshii) [reply wiki/osu!supporter/en.md:2]: I'll change that as soon as I'm at home

**Comment by geshii** [comment]
I think we're good now

**Review by TicClick — APPROVED** [review]
alright

File: wiki/osu!supporter/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/osu!supporter/en.md:2]: reading over the article some sentences sound odd due to mixing formal and casual style, and the author sounds uncertain about details, which is really strange for a wiki article. I imagine all I meant by my TODO comment was that it needed a quick pass by a native speaker. if these are too nitpicky of reasons for `needs_cleanup` and/or my comment, feel free to remove

Comment (TicClick) [reply wiki/osu!supporter/en.md:2]: they aren't, thanks for clarification

**Pull request merged**